artifacts/rhdh-plugins-reference/quay/quay-plugin-readme.adoc

```diff
@@ -105,7 +105,7 @@ Quay is a front-end plugin that enables you to view the information about the co
 .Prerequisites
 
 * Your {product-short} application is installed and running.
-* You have installed the Quay plugin. For installation process, see <<installation,Installation>>.
```

Comment:
Rather than removing the links, I’d suggest keeping them and ensuring they follow a clear structure that aligns with our migration strategy, which we may need to address in the future. Please feel free to create a Jira for the link improvement effort if needed.